Update TextBox description (#492)
DevExpress/devextreme-demos: JSDemos/Demos/TextBox/Overview/description.md

The [TextBox](/Documentation/ApiReference/UI_Widgets/dxTextBox/) is a UI component that allows users to enter and edit a single line of text. This demo illustrates the following  **TextBox** options:

- [value](/Documentation/ApiReference/UI_Widgets/dxTextBox/Configuration/#value) 
A value the **TextBox** displays.
- [placeholder](/Documentation/ApiReference/UI_Widgets/dxTextBox/Configuration/#placeholder)       

> A default text in the **TextBox**, displayed when its value is not defined.

Reviewer: ```suggestion
A text that the **TextBox** displays when the **value** is not defined.
```

- [showClearButton](/Documentation/ApiReference/UI_Widgets/dxTextBox/Configuration/#showClearButton)        

> Determines whether the Clear button is displayed. The button is used to clear the **TextBox** value.

Reviewer: ```suggestion
Specifies whether to display the button that clears the **TextBox** **value**.
```

> - [mode (password in this demo)](/Documentation/ApiReference/UI_Widgets/dxTextBox/Configuration/#mode)        

Reviewer: ```suggestion
- [mode](/Documentation/ApiReference/UI_Widgets/dxTextBox/Configuration/#mode)        
```

> This property affects a set of characters displayed on a mobile device's keyboard and adds certain visual features to the UI component. In this example, characters are obscured so that the password cannot be read.

Reviewer: ```suggestion
Affects a set of keyboard characters displayed on a mobile device when the **TextBox** gets focus and adds specific visual features to the UI component. In this example, the **mode** is set to *"password"* so that entered characters are obscured, and the password cannot be read.
```

- [mask](/Documentation/ApiReference/UI_Widgets/dxTextBox/Configuration/#mask)        

> A custom format for the entered string.

Reviewer: ```suggestion
A mask that specifies a custom format for the entered string.
```

- [maskRules](/Documentation/ApiReference/UI_Widgets/dxTextBox/Configuration/#maskRules)        

> Use this parameter to customize mask rules.

Reviewer: ```suggestion
Custom mask rules.
```

- [disabled](/Documentation/ApiReference/UI_Widgets/dxTextBox/Configuration/#disabled)        
Specifies whether the TextBox responds to user interaction.
- [onValueChanged](/Documentation/ApiReference/UI_Widgets/dxTextBox/Configuration/#onValueChanged) event handler      

> Use this function to perform an action when a user chooses a new value. In this demo, this function is used to display the selected value.

Reviewer: `Was` this just copied from the SelectBox description?

```suggestion
Use this function to perform an action when a user enters a new value. In this demo, this function uses the entered value to combine a dummy email address and assigns it a **value** to another **TextBox**.
```

- [valueChangeEvent](/Documentation/ApiReference/UI_Widgets/dxTextBox/Configuration/#valueChangeEvent) 

> Use this parameter to declare DOM events that trigger an update of the UI component's value.

Reviewer: ```suggestion
One or several DOM events that trigger the **onValueChanged** event handler.
```

- [readOnly](/Documentation/ApiReference/UI_Widgets/dxTextBox/Configuration/#readOnly)     

> Prevents users from changing the editor's value via the UI.

Reviewer: ```suggestion
Prevents users from changing the editor's value.
```

- [hoverStateEnabled](/Documentation/ApiReference/UI_Widgets/dxTextBox/Configuration/#hoverStateEnabled)        

> Allows users to enable or disable **TextBox** state change on hover.

Reviewer: ```suggestion
Specifies whether the **TextBox** responses when users long press or hover the mouse pointer over it.
```

> The [TextBox](/Documentation/ApiReference/UI_Widgets/dxTextBox/) is a UI component that allows users to enter and edit a single line of text. This demo illustrates the following  **TextBox** options:

Reviewer: ```suggestion
The [TextBox](/Documentation/ApiReference/UI_Widgets/dxTextBox/) is a UI component that allows users to enter and edit a single line of text. This demo illustrates the following  **TextBox** properties:
```

- [value](/Documentation/ApiReference/UI_Widgets/dxTextBox/Configuration/#value)     

> A value the **TextBox** displays.

Reviewer 2: ```suggestion
The **TextBox** value.
```

- [placeholder](/Documentation/ApiReference/UI_Widgets/dxTextBox/Configuration/#placeholder)       

> A text that the **TextBox** displays when the **value** is not defined.

Reviewer 3: I'd rather call it an "input prompt" than the "text the editor displays". 

```suggestion
An input prompt the **TextBox** displays when the **value** is not defined.
```

- [showClearButton](/Documentation/ApiReference/UI_Widgets/dxTextBox/Configuration/#showClearButton)        
Specifies whether to display the button that clears the **TextBox** **value**.

- [mode](/Documentation/ApiReference/UI_Widgets/dxTextBox/Configuration/#mode)        

> Affects a set of keyboard characters displayed on a mobile device when the **TextBox** gets focus and adds specific visual features to the UI component. In this example, the **mode** is set to *"password"* so that entered characters are obscured, and the password cannot be read.

Reviewer 3: I'd change "specific features". The word "specific" doesn't add much value, I think. So maybe something like this:

```suggestion
Affects a set of keyboard characters displayed on a mobile device when the **TextBox** gets focus and modifies the UI component's display style. In this example, the **mode** is set to *"password"* so that entered characters are obscured, and the password cannot be read.
```

> Affects a set of keyboard characters displayed on a mobile device when the **TextBox** gets focus and modifies the UI component's display style. In this example, the **mode** is set to *"password"* so that entered characters are obscured, and the password cannot be read.

Reviewer 2: ```suggestion
Affects a set of keyboard characters displayed on a mobile device when the **TextBox** gets focus and modifies the UI component's display style. In this example, the **mode** is set to *"password"* to obscure entered characters, and the password is not shown.
```

- [mask](/Documentation/ApiReference/UI_Widgets/dxTextBox/Configuration/#mask)        

> A mask that specifies a custom format for the entered string.

Reviewer 3: I think the meaning of "mask" is well-known. 

```suggestion
An input mask.
```

- [maskRules](/Documentation/ApiReference/UI_Widgets/dxTextBox/Configuration/#maskRules)        
Custom mask rules.

- [disabled](/Documentation/ApiReference/UI_Widgets/dxTextBox/Configuration/#disabled)        
Specifies whether the TextBox responds to user interaction.

- [onValueChanged](/Documentation/ApiReference/UI_Widgets/dxTextBox/Configuration/#onValueChanged) event handler      

> Use this function to perform an action when a user enters a new value. In this demo, this function uses the entered value to combine a dummy email address and assigns it a **value** to another **TextBox**.

Reviewer 3: ```suggestion
Use this function to perform an action when a user enters a new value. In this demo, this function uses the entered value to construct a dummy email address and assign it to another **TextBox**.
```

- [valueChangeEvent](/Documentation/ApiReference/UI_Widgets/dxTextBox/Configuration/#valueChangeEvent)     
One or several DOM events that trigger the **onValueChanged** event handler.

- [readOnly](/Documentation/ApiReference/UI_Widgets/dxTextBox/Configuration/#readOnly)     
Prevents users from changing the editor's value.

- [hoverStateEnabled](/Documentation/ApiReference/UI_Widgets/dxTextBox/Configuration/#hoverStateEnabled)        

> Specifies whether the **TextBox** responses when users long press or hover the mouse pointer over it.

Reviewer: ```suggestion
Specifies whether the **TextBox** responds when users long press or hover the mouse pointer over it.
```
